### PR TITLE
merged

### PR DESCRIPTION
**Pull request by valnav**
the change is that i have removed support for test requests for upload certificate and upload pkcs. instead now, you have to pass the id and a valid path to the cert or pfx file

### CODE REVIEW

File: PolicyAndKeys-Client/Program.cs [hunk 1]
```diff
@@ -189,11 +206,15 @@ private static string[] ProcessParametersInput()
                     Console.WriteLine($"For Command: {cmdType.ToString()} specify path of {resType.ToString()} ");
                     break;
                 case CommandType.UPDATE:
+                case CommandType.UPLOADSECRET:
+
+                    Console.WriteLine($"For Command: {cmdType.ToString()} (space separated) specify Id and path of {resType.ToString()} ");
+                    if (resType != ResourceType.policies) Console.WriteLine("optionally, type test for both, if you want to simply try this out");
+                    break;
+
                 case CommandType.UPLOADCERTIFICATE:
```
Comment (agrabhi) [inline PolicyAndKeys-Client/Program.cs:222]: Can we make sure that GenerateKey, uploadcert, uploadpkcs12, uploadcert, getactive key dont work for policies.

File: PolicyAndKeys-Lib/Constants.cs [hunk 2]
```diff
@@ -46,7 +46,7 @@ public class Constants
 
         public static string UploadCertificate = @"{{  'key': '{0}' }} ";
 
-        public static string UploadPkcs = @"{{  'key': '{0}',   'password': '{0}' }}";
+        public static string UploadPkcs = @"{{  'key': '{0}',   'password': 'password' }}";
```
Comment (agrabhi) [inline PolicyAndKeys-Lib/Constants.cs:49]: this should be {1}

File: PolicyAndKeys-Lib/TestRequests.cs [hunk 3]
```diff
@@ -82,14 +82,20 @@ public bool CheckAndGenerateTest(ref string id, ref string content)
...
+        }
         private void CheckBeforeCreateTestKeyset()
         {
             if (TestKeysetID == null)
             {
                 var json = Constants.CreateKeyset;
-                var guid = Guid.NewGuid().ToString();
 
-                json = json.Replace(Constants.KEYSETID_TOKEN, guid);
+                json = json.Replace(Constants.KEYSETID_TOKEN, RandomString(8));
```
Comment (agrabhi) [inline PolicyAndKeys-Lib/TestRequests.cs:98]: WE could have just got an input name from the customer

**Review by agrabhi — CHANGES_REQUESTED** [review]
Need some changes

File: PolicyAndKeys-Client/Program.cs (hunk 1 above)
Comment (valnav) [reply PolicyAndKeys-Client/Program.cs:222]: yes they wont, because the resource type is different

File: PolicyAndKeys-Lib/Constants.cs (hunk 2 above)
Comment (valnav) [reply PolicyAndKeys-Lib/Constants.cs:49]: updated.

File: PolicyAndKeys-Lib/TestRequests.cs (hunk 3 above)
Comment (valnav) [reply PolicyAndKeys-Lib/TestRequests.cs:98]: i know i am trying to make it easier for somebody to just test it.

**Pull request merged**